djangocms_versioning/test_utils/factories.py

```diff
@@ -25,6 +25,13 @@ class UserFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = User
 
+    @classmethod
+    def _create(cls, model_class, *args, **kwargs):
```

Comment:
I don't like overriding private methods but if this is the only way for overriding in the factory then it's ok.